### ISSUE TITLE
Studio config updates

### ISSUE THREAD

**Issue by dberenbaum** [iterative/vscode-dvc#3898]
There are a few updates needed for the Studio config:
1. After https://github.com/iterative/dvc/pull/9433, live sharing will be enabled by default when the studio token is saved to the dvc config, although it can be disabled with `studio.offline = true`. To comply with this, we need to make the "Share new experiments live" checkbox correspond to the `studio.offline` config value and stop setting the environment variable.
2. Since all these settings can be set at various levels (system/user/project/local config or environment variable), we need to make clear where it's being set and link to open the config file. Ideally, we would allow users to choose the level where they want it set. 

<img width="951" alt="Screenshot 2023-05-15 at 4 54 15 PM" src="https://github.com/iterative/vscode-dvc/assets/2308172/92975ca5-f3b8-4572-aa93-380025e6cbf1">

Related: #3574

**Comment by mattseddon** [comment]
> Since all these settings can be set at various levels (system/user/project/local config or environment variable), we need to make clear where it's being set and link to open the config file. Ideally, we would allow users to choose the level where they want it set.

When managing `studio.token` through the extension we set the value at the user level (`--global`). We currently do not expose the complexity of 4 separate configs to users. We also do not give the option to open the config files. However, if they choose to save the token using `--system`, `--project` or `--local` we respect the value. When trying to remove the value (from the disconnect action) we only attempt to remove `studio.token` from the `--local`/`--global` configs. I built the functionality in this way because it should support 90% of people being onboarded. IMO the remaining 10% should be capable of managing their configs themselves.
